app/views/diary_entries/_diary_entry.html.erb

```diff
@@ -44,4 +44,10 @@
       <% end %>
     </ul>
   </nav>
+
```

Comment:
The buttons ate too close to the actions list above. Probably need to add margins to it.
![image](https://github.com/user-attachments/assets/e5f28f53-ca9d-4642-b463-46e2c4d87daf)